db/migrate/20200811003516_create_participating_sellers.rb

```diff
@@ -0,0 +1,12 @@
+class CreateParticipatingSellers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :participating_sellers do |t|
+      t.string :name
+      t.bigint :seller_id, null: true
```

Comment:
actually I just realized, I think null defaults to true, should the other options (name stamp_url etc) be set to null:false instead?

Reply:
sure, makes sense to me, whatever's default can be default, no need to manually specify